tools/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -311,29 +301,68 @@ public partial class OutlineService : ServiceBase {
     public void ResetRouting(string proxyIp, string proxyInterfaceName) {
       // Proxy server.
       if (proxyIp != null) {
-        try {
-          DeleteProxyRoute(proxyIp, proxyInterfaceName);
-        } catch (Exception e) {
-          eventLog.WriteEntry($"failed to remove route to proxy server: {e.Message}", EventLogEntryType.Error);
-        }
+        DeleteProxyRoute(proxyIp, proxyInterfaceName);

```

Comment:
It looks like this function can still raise an exception. Did you want to exit now if that happens?

Reply:
Added a try/catch.